rpm/libmlocale-qt5.spec

```diff
@@ -52,7 +52,9 @@ export QT_SELECT=5
 
 %install
 export QT_SELECT=5
-%make_install
+%qmake5_install
```

Comment:
%make_build but %qmake5_install feels asymmetrical?

Reply:
A bit pondering if in principle such %qmake5_install (or build) should even exist. Qmake builds are done by executing qmake which generates makefiles and the actual build and installation are done with make. For comparison there is no %automake_build or %automake_install.

Reply:
Please check the command that is executed in %qmake_install and how qmake defines things such as DESTDIR.
Qmake doesn't use the standard variable DESTDIR in `make install` but their own `INSTALL_ROOT` .
The `%qmake5_install` macro is there to handle these or other differences when needed, only the former is done at this moment.